src/lib/logger.ts

```diff
@@ -99,10 +113,6 @@ export default class Logger {
       time: Date.now(),
     };
     WalletStorage.set(STORAGE_KEYS.errorLog, [...errorLog, logEntry]);
```

Comment:
Idea was also to collect data only for last month/week (better consult it).

Reply:
Agreed to keep the last 1000 entries. Updated the code and wording of the Error Log Settings page.